nixie/unittests/test_verbose.py

```
@@ -0,0 +1,77 @@
+import logging
+import shlex
+import shutil
+import sys
+from pathlib import Path
+
+import pytest
+
+from nixie.cli import get_mmdc_cmd, parse_args, render_block
+
+
+def test_parse_args_verbose(monkeypatch) -> None:
```

Ruff (0.12.2)

Missing type annotation for function argument `monkeypatch`

(ANN001)